quora-api/src/main/java/com/upgrad/quora/api/controller/QuestionController.java

```diff
@@ -29,7 +29,7 @@
    * @return UUID of the question created in DB.
    * @throws AuthorizationFailedException In case the access token is invalid.
    */
-  @RequestMapping(
+  @PostMapping(
       method = RequestMethod.POST,
```

Comment:
remove this line, as @PostMapping annotation means method is mapped to HTTP post request